app/src/main/java/org/mozilla/reference/browser/library/history/ui/HistoryFragment.kt

```diff
@@ -93,8 +95,12 @@ class HistoryFragment : Fragment(), UserInteractionHandler {
     }
 
     private fun openHistoryItem(item: HistoryItem) {
-        historyViewModel.openHistoryItem(item)
-        onBackPressed()
+        (context as BrowserActivity).openToBrowserAndLoad(
+            searchTermOrUrl = item.url,
+            newTab = false,
+            from = BrowserDirection.FromHistory,
+            private = false
+        )
```

Comment:
This can be done with extension methods on `Context` or `Activity`:

```
fun Activity.openToBrowserAndLoad(....) = (this as? BrowserActivity)?.openToBrowserAndLoad(...)
```